src/visualization/vega_renderer/JSCanvas.m

```diff
@@ -694,13 +694,12 @@ - (VegaCGLinearGradient *)createLinearGradientWithX0:(double)x0 y0:(double)y0 x1
 @implementation VegaCGCanvas
 
 - (instancetype)initWithContext:(CGContextRef)parentContext {
-    self = [super init];
+    self = [super initWithTagName:@"canvas"];
```

Comment:
As we're setting the `context` property on `self` it might be a good idea to check if the call to `super` returned non-nil, as in [here](https://github.com/apple/turicreate/blob/bca0270cc1090c5c9c8b0a3526d2d4c756a23cdf/src/ml/neural_net/TCMPSImageAugmenting.m#L33).

Reply:
Good catch. There are a few other spots where I need to add this as well. I'll push an update.